src/multi-selector.js

```diff
@@ -426,7 +426,7 @@ export default class MultiSelector extends React.Component {
     let pills = this.state.selectedItems
 			.map((item, i) => {
 				return (
-					<Pill key={i} item={item} removeItem={partial(this.removeItem, item)} color={this.props.color} getItemTitle={getItemTitle}/>
+					<Pill key={item.id || uniqueId()} item={item} removeItem={partial(this.removeItem, item)} color={this.props.color} getItemTitle={getItemTitle}/>
```

Comment:
won't this cause a new uniqueId to be given (assuming there's no `item.id`) on every render? Which would mean unnecessary unmounting of the old one, and construction and rendering of the new one?

i.e. random numbers as keys are bad, right?

Reply:
Oooo, interesting! I wasn't thinking about that.
https://stackoverflow.com/questions/29808636/when-giving-unique-keys-to-components-is-it-okay-to-use-math-random-for-gener

Reply:
I did some testing. It looks as though when I update the list the only one that is unmounted is the one that I am removing. I didn't see any others unmount unnecesarily

Reply:
You're right. I'll fix that